### ISSUE TITLE
Automatisierung der Terminverwaltung

### ISSUE THREAD

**Issue by PMudra** [PG-MATE/Organisation#33]
Fast alle Punkte der Verwaltung könnten automatisiert werden und so viel Zeit über das ganze Projekt gespart werden. Das Erstellen dieser Automatisierung sollte jedoch nicht zu kompliziert werden.
Einige Ideen:

- [ ] Erstellung des Meilensteins für den Termin
- [ ] Erstellung des Moderations und Protokollanten Tasks
- [ ] Zuweisung der entsprechenden Beauftragten
- [ ] Anlegen des Protokolls
- [ ] Bearbeitung der README.md der Hauptseite
- [ ] Erstellung der Einladungsmail
- [ ] Verschicken der Einladungsmail

Umzusetzen wohl am besten mit einer Skriptsprache (wie zb JavaScript) für die es eine GitHub Api gibt.

**Comment by SebastianSchmidt** [comment]
Das wäre vielleicht eine tolle Gelegenheit, an einem "echten" Projekt gemeinsam Test-driven vorzugehen.

**Comment by PMudra** [comment]
Also wenn es jemand üben möchte, ja. Allerdings würde ich aus der Sache kein offizielles "Sub-Projekt" machen, da es sich in meinen Augen nur um einen Helfers-Helfer Skript handelt. Davon sollten im Verlauf des Projekts möglichst überall da welche entstehen, wo es sinnvoll ist.

**Comment by SebastianSchmidt** [comment]
Genau, ich würde das auch nicht als "offizielles Projekt" betrachten, aber schon ein eigenes Repository anlegen. Vielleicht könnte man das ganze als Pair Programming durchführen, um das schon mal innerhalb der Gruppe zu üben.

**Comment by PMudra** [comment]
Ganz coole Anleitung für die Erstellung eines node.js command line tools
https://developer.atlassian.com/blog/2015/11/scripting-with-node/

**Comment by SebastianSchmidt** [comment]
Ich habe mir mal die GitHub-APIs für das Erstellen und Verwalten von [Issues](https://developer.github.com/v3/issues/) und [Milestones](https://developer.github.com/v3/issues/milestones/) angeschaut. Zusammen mit der [Anleitung](https://developer.atlassian.com/blog/2015/11/scripting-with-node/), die @PMudra gefunden hat, dürfte eine Umsetzung recht schnell möglich sein.
